demo_nodes_cpp/CMakeLists.txt

```diff
@@ -92,9 +92,12 @@ function(create_demo_library plugin executable)
     ARCHIVE DESTINATION lib
     LIBRARY DESTINATION lib
     RUNTIME DESTINATION bin)
+  SET(DEMO_EXECUTOR "rclcpp::executors::SingleThreadedExecutor" CACHE STRING "The executor for the demo nodes")
```

Comment:
Out of curiosity, have you tried this with the `MultiThreadedExecutor`.  I'm not sure that all of our demo nodes are written to work correctly under that executor (but would be interesting to see).

Reply:
I gave this a spin (no pun intended) and it seems to work okay. I guess most of the demo nodes have a single callback group so there shouldn't be a difference?